Sources/Orbit/Components/ListChoiceGroup.swift

```diff
@@ -1,5 +1,14 @@
 import SwiftUI
 
+public enum ListChoiceGroupStyle {
+    
+    // Style with no decoration.
+    case borderless
+    
+    // Style with Card-like appearance.
+    case card(status: Status? = nil, backgroundColor: Color? = .clear)
```

Comment:
Default value here should be `nil` in case of optionals - if we further use it to distinguish "no change, let caller decide later". I believe `nil` it will later default to `.white` which is probably what we want in the end?

Reply:
`Card` has
```
backgroundColor: Color? = .white
```
in all its initializers - should we change those too?